picard_plugin_tools/__init__.py

```diff
@@ -1,23 +1,67 @@
 import ast
-import os
 import json
+import os
+import re
 import zipfile
 
 from hashlib import md5
 
+import click
```

Comment:
This new dependancy should at least appear in a `requirements.txt` file.

And, since `click` module is used, please add a `main()` and allows the module to be called with command line options.